src/plugins/MarkdownShortcuts.js

```diff
@@ -118,16 +120,25 @@ export default function MarkdownShortcuts() {
 
       // if we have multiple tags then mark the text between
       if (inlineTags.length > 1) {
-        const firstText = startBlock.getFirstText();
+        const firstText = focusText;
         const firstCodeTagIndex = inlineTags[0];
         const lastCodeTagIndex = inlineTags[inlineTags.length - 1];
+
+        const tokenLen = lastCodeTagIndex - firstCodeTagIndex - shortcut.length;
+
+        ev.preventDefault();
+        // Add space character at end of the text so it doesn't get wrapped
+        // within the node containing the mark we're applying.
+        editor.insertText(' ');
+
         return editor
           .removeTextByKey(firstText.key, lastCodeTagIndex, shortcut.length)
           .removeTextByKey(firstText.key, firstCodeTagIndex, shortcut.length)
           .moveAnchorTo(firstCodeTagIndex, lastCodeTagIndex - shortcut.length)
```

Comment:
I'm pretty sure that moveAnchorTo only takes one argument and that this second argument isn't actually doing anything. At least I can't find any documentation for a second argument.